Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_impl.cs

```diff
@@ -929,7 +929,6 @@ public partial class PictureFlusher {
         }
 
         void _drawTextBlob(TextBlob textBlob, uiOffset offset, uiPaint paint) {
-            D.assert(textBlob != null);
```

Comment:
why textBlob can be null ?

Reply:
Changed to struct. Not sure how to check if it is initialized.